source/includes/mongodb-5.0-required-for-sync-admonition.rst

```diff
@@ -1,5 +1,5 @@
-.. note:: MongoDB Version 4.4 or Higher Required
+.. note:: MongoDB Version 5.0 or Higher Required
    
-   In order to use Atlas Device Sync, your Atlas cluster must use MongoDB version 4.4 or higher.
-   When setting up your cluster, select :guilabel:`MongoDB 4.4` (or a higher version)
+   In order to use Atlas Device Sync, your Atlas cluster must use MongoDB version 5.0 or higher.
+   When setting up your cluster, select :guilabel:`MongoDB 5.0` (or a higher version)
```

Comment:
Some minor edits for the pre-existing content. Feel free to ignore if you want to just merge this.

```suggestion
   Atlas Device Sync requires an Atlas cluster running MongoDB version 5.0 or later.
   When setting up your cluster, select :guilabel:`MongoDB 5.0` (or a later version)
```